### ISSUE TITLE
[Bug]: Normalised IP metric bigger than one?

### ISSUE THREAD

**Issue by mrceresa** [milvus-io/pymilvus#1204]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Some of the searches return an IP metric bigger than one. This puzzles me, as it should be [-1,1] AFAIK.

They are only slightly out of range  (i.e. 1.00035381317139, 1.00012695789337) so, initially I thought about a rounding error, but later on I thought that maybe I somehow got the normalisation wrong?

This is the collection schema and index definition:

~~~
all_fields = [
        FieldSchema(name="pk", dtype=DataType.INT64, is_primary=True, auto_id = auto_id),
        FieldSchema(name="embedding", dtype=DataType.FLOAT_VECTOR, dim=dim)
        ]

index_params = {
    'index_type': 'IVF_SQ8',
    'params': {'nlist': 512},
    'metric_type': 'IP'
    }
~~~

and when I load the vectors I create them and normalise like this:
~~~
Y = model.transform(X).toarray()
embeddings = list(sklearn.preprocessing.normalize(Y))
~~~

### Expected Behavior

The returned distances using IP metric should be [-1,1]

### Environment details

```markdown
- Method of installation (Docker, or from source): docker milvusdb/milvus:v2.1.2
- Milvus version (v0.3.1, or v0.4.0): not sure, I can't see it from the docker
- Milvus configuration (Settings you made in `server_config.yaml`):
```

### Anything else?

Thanks a lot for the great software!!!
